orion-server/src/main/java/com/pinterest/orion/core/actions/aws/TerminateEC2InstanceAction.java

```diff
@@ -73,6 +75,13 @@ public void runAction() {
         );
         getEngine().alert(AlertLevel.MEDIUM, msg);
         getEngine().alert(AlertLevel.HIGH, msg);
```

Comment:
Unrelated to this diff. Wondering why there are two different alert levels here.

Reply:
https://github.com/pinterest/orion/blob/master/orion-server/src/main/java/com/pinterest/orion/core/actions/alert/AlertFactory.java#L43

The alertFactory only triggers the type of alerts with same level. For example, pagerduty alert is high level and email alert is low level. It alerts twice with different levels just to trigger multiple types of alerts.